src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const data = [...document.querySelectorAll('.population')];
+
+const populations = data.map((value) => {
+  const text = value.innerText.trim();
+  const numericText = text.split(',').join('');
+
+  return Number(numericText);
+});
+
+const total = populations.reduce((sum, value) => sum + value, 0);
+const average = Math.round(total / populations.length);
```

Comment:
Ensure that 'populations.length' is not zero to prevent division by zero errors when calculating the average.